process/smartContract/process.go

```diff
@@ -807,6 +807,8 @@ func (sc *scProcessor) ExecuteBuiltInFunction(
 			return vmcommon.ExecutionFailed, sc.ProcessIfError(acntSnd, txHash, tx, err.Error(), []byte("gas consumed exceeded"), snapshot, vmInput.GasLocked)
 		}
 
+		sc.penalizeUserIfNeeded(tx, txHash, newVMInput.CallType, newVMInput.GasProvided, newVMOutput)
```

Comment:
Builtin functions are already activated. Is this backwards compatibile?

Reply:
done